specs-go/config.go

```diff
@@ -615,6 +615,26 @@ type Arch string
 // LinuxSeccompFlag is a flag to pass to seccomp(2).
 type LinuxSeccompFlag string
 
+const (
+	// LinuxSeccompFlagLog is a seccomp flag to request all returned
+	// actions except SECCOMP_RET_ALLOW to be logged. An administrator may
+	// override this filter flag by preventing specific actions from being
+	// logged via the /proc/sys/kernel/seccomp/actions_logged file. (since
+	// Linux 4.14)
+	LinuxSeccompFlagLog LinuxSeccompFlag = "SECCOMP_FILTER_FLAG_LOG"
+
+	// LinuxSeccompFlagTsync is the filter to synchronize all other threads of
+	// the calling process to the same seccomp filter tree. A "filter tree" is
+	// the ordered list of filters attached to a thread. (Attaching identical
+	// filters in separate seccomp() calls results in different filters from
+	// this perspective.)
```

Comment:
As I pointed out in another PR doing the same, I don't think we should add a const for this: https://github.com/opencontainers/runtime-spec/pull/1108#discussion_r680080483